tools/hitmap/hitmap.xml

```diff
@@ -0,0 +1,297 @@
+
+<tool id="hitmap" name="HiT-MaP" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" license="MIT">
+    <description>High-resolution Maldi-imaging Proteomics Analysis</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <edam_topics>
+        <edam_topic>topic_0121</edam_topic>
+    </edam_topics>
+
+    <edam_operations>
+        <edam_operation>operation_3630</edam_operation>
+    </edam_operations>
+
+    <requirements>
+        <container type="docker">mashuoa/hitmap@sha256:97f2f853362b56be6c2b0fb1507e0288667e7fb6ad0eda3ff1d761dd72dfaad0</container>   
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir expdata &&
+        cp '${file_inputs.imzml_file}' expdata/sample.imzML &&
```

Comment:
is `ln` enough?

Reply:
Unfortunatenly not. Hitmap needs the file extension to be intact, but all galaxy data has extension '.dat'

Reply:
But instead of using `cp` you can do `ln -l '${file_inputs.imzml_file}' expdata/sample.imzML`

That saves a copy operation. And works in 95% of all cases. In some cases, the tool does not follow symlinks ... I hope this is not the case here.

Reply:
Mmm gave this a shot, but since it's a container, it was having difficulty creating the link. 

I think this is an improvement which could appear in a later version of the wrapper. 

HiTMaP has been very temperamental and hard to get working, so I'd rather just keep as is rather that spending time on this optimization.